src/scripting/flash/system/flashsystem.cpp

```diff
@@ -139,6 +159,11 @@ ASFUNCTIONBODY_ATOM(Capabilities,_getServerString)
 		res += buf;
 	}
 
+    // Add hasAudio and hasMP3 to ServerString if available
+#ifdef ENABLE_LIBAVCODEC
+    res += "&A=t&MP3=t";
+#endif
```

Comment:
"A=t" and "MP3=t" are already in the default server string (see a few lines above) , I think we should remove them from there.
When that is done, we can merge this